sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubRemoteCredentialUserData.java

```diff
@@ -0,0 +1,22 @@
+package com.mercedesbenz.sechub.commons.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SecHubRemoteCredentialUserData {
+
+    public static final String PROPERTY_NAME = "name";
+    public static final String PROPERTY_PASSWORD = "password";
+
+    private String name;
```

Comment:
We should add setters for name and password.
Why ?
Because e.g. in system tests we create the config programmatically. Also unit tests cannot create the data object by themself etc.